src/Api.Rest.Dtos/Data/ServiceInformationDto.cs

```diff
@@ -44,6 +44,13 @@ public class ServiceInformationDto
 		[JsonPropertyName( "version" )]
 		public string Version { get; set; }
 
+		/// <summary>
+		/// Gets or sets the extended server version.

```

Comment:
Could you please use the same text in both summaries? The second one is slightly different 😀
I would vote for 'Gets or sets the extended server version'